src/ddsketch/mapping/KeyMapping.ts

```diff
@@ -78,6 +78,7 @@ export class KeyMapping implements Mapping {
     }
 
     toProto(): IIndexMapping {
+        const ProtoIndexMapping = require('../proto/compiled').IndexMapping;
```

Comment:
There are still some static `import`s to `/proto/compiled` at the top of this file, does that matter? I would sort of expect that any one of those would cause the module missing error, regardless of if an end user is invoking `toProto` / `fromProto`.

Reply:
It seems that the TypeScript compiler is able to understand that actual code isn't used and not add the `require` call.